content/en/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -203,6 +203,13 @@ following:
     EOF
     ```
 
+    {{< note >}}
+    `pod-security.admission.config.k8s.io/v1` configuration requires v1.25+.
+    For v1.23 and v1.24, use [v1beta1](https://v1-24.docs.kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-admission-controller/).
+    For v1.22, use [v1alpha1](https://v1-22.docs.kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-admission-controller/).
+    {{< /note >}}
```

Comment:
Do we just want to copy this wording instead? https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/security/pod-security-admission.md#built-in-pod-security-admission-enforcement

> This page is part of the documentation for Kubernetes v{{< skew currentVersion >}}. If you are running a different version of Kubernetes, consult the documentation for that release.

Reply:
eh... if we have nice links, I'd rather give them those